### DISCUSSION TITLE
IOS设备连不上？

### DISCUSSION THREAD

**Discussion by Revisit1** [getlantern/lantern-forum-cn#984] [General]
开了专业版，手机和iPad就没连上过，什么情况？报告问题也发送不了
![DC87632E-C44E-4786-868D-6FF0BE532916](https://github.com/user-attachments/assets/60e173ea-5bc3-4d53-9b12-d36409c6954b)
![B491722C-7487-49F4-995B-1A5184A4E0B1](https://github.com/user-attachments/assets/7af0db1f-029d-4d12-a087-e5619ac4f2ee)

**Comment by Revisit1** [comment]
![433751A1-2903-48E4-A984-F9410B76B403](https://github.com/user-attachments/assets/320cfb75-fc59-4a21-9c9a-e77e2eccf082)

**Comment by MaxMKMax** [comment]
@Revisit1 您好. 请问您用的是最新版本吗? 如果可以的话您再试试看可以帮我提交各应用内报告吗? 试试几次看看. 

同时我也会帮您反馈上去. 还有, 请问您连接到的是什么服务器?

非常抱歉影响了您的体验感. 我们会尽快解决问题的. 谢谢您的耐心等待.

**Reply by Revisit1** [reply]
是最新版；已经反复试过，应用内报告还是无法提交；换了很多节点，貌似都不太行
![F04E00AE-C129-4259-8668-075DAFA18BB7](https://github.com/user-attachments/assets/5a920cab-7c37-45bd-bd75-d2257568a3f7)
![D71BAB7B-B64D-4C34-A6C3-6D34976D27BD](https://github.com/user-attachments/assets/e96d4254-0aac-4e42-8fae-9f8efb89469f)
![0DF336FC-90E5-4EA8-8C31-312E591FCFD7](https://github.com/user-attachments/assets/55da0774-483d-48df-903c-e23197a524ac)
![722784B6-7DE7-4118-872B-5B47EA096685](https://github.com/user-attachments/assets/856466e3-dd2c-467f-9c1c-beeb8b30ceb0)
![E065F325-8DF9-40A2-9AC2-7747178E531C](https://github.com/user-attachments/assets/296d1fc8-b4ed-439f-b9a0-39b540d4dd61)

**Comment by MaxMKMax** [comment]
@Revisit1 您好. 明白了. 我再帮您反馈一下. 下周二晚上和技术团队开会的时候开会我帮您备注一下. 不好意思啊. 谢谢您支持蓝灯!
